src/ApiService/ApiService/onefuzzlib/Reports.cs

```diff
@@ -28,9 +28,9 @@ public class Reports : IReports {
 
     public async Async.Task<IReport?> GetReportOrRegression(Container container, string fileName, bool expectReports = false, params string[] args) {
         var filePath = string.Join("/", new[] { container.String, fileName });
-        if (!fileName.EndsWith(".json", StringComparison.Ordinal)) {
+        if (!fileName.EndsWith(".json", StringComparison.Ordinal) || fileName.Contains("source-coverage", StringComparison.Ordinal)) {
```

Comment:
I think `StringComparison.InvariantCultureIgnoreCase` is the recommended option: https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#specifying-string-comparisons-explicitly

Reply:
Ah good to know, thank you!

Reply:
For machine-readable stuff like this filename `OrdinalIgnoreCase` is more suited. The culture-sensitive versions can be _very_ weird if you're not expecting it:

```csharp
"source-coverage".Contains("C­", StringComparison.InvariantCultureIgnoreCase)
```